assets/src/scss/new-identity/style.scss

```diff
@@ -42,38 +42,67 @@
   --button-secondary--hover--background: var(--p4-dark-green-800);
   --button-secondary--hover--border-color: var(--p4-dark-green-800);
   --button-secondary--hover--color: var(--white);
+  --table-header-grey--background: var(--grey-800);
+  --table-odd-row-grey--background: var(--grey-200);
+  --table-even-row-grey--background: var(--grey-100);
+  --table-footer-grey--background: var(--grey-200);
+  --table-header-green--background: var(--p4-dark-green-800);
+  --table-odd-row-green--background: var(--beige-200);
+  --table-even-row-green--background: var(--beige-100);
+  --table-footer-green--background: var(--beige-200);
+  --table-header-blue--background: #167f82;
+  --table-odd-row-blue--background: var(--beige-200);
+  --table-even-row-blue--background: var(--beige-100);
+  --table-footer-blue--background: var(--beige-200);
+  --table-header-gp-green--background: var(--gp-green-800);
+  --table-odd-row-gp-green--background: var(--beige-200);
+  --table-even-row-gp-green--background: var(--beige-100);
+  --table-footer-gp-green--background: var(--beige-200);
   --block-spreadsheet--color: var(--grey-900);
-  --block-spreadsheet-grey-header--background: var(--grey-800);
-  --block-spreadsheet-grey-odd-row--background: #eaedee;
-  --block-spreadsheet-grey-even-row--background: var(--grey-100);
-  --block-spreadsheet-green-header--background: var(--gp-green-800);
-  --block-spreadsheet-green-odd-row--background: var(--beige-200);
-  --block-spreadsheet-green-even-row--background: var(--beige-100);
-  --block-spreadsheet-blue-header--background: #167f82;
-  --block-spreadsheet-blue-odd-row--background: var(--beige-200);
-  --block-spreadsheet-blue-even-row--background: var(--beige-100);
   --link--color: var(--grey-900);
   --link--visited--color: var(--grey-900);
   --link--hover--color: var(--grey-900);
 }
 
 // Spreadsheet block new color.
 table.spreadsheet-table.is-color-dark-green {
-  th --block-spreadsheet-dark-green-header-- {
-    background: var(--p4-dark-green-800);
+  th {
+    background-color: var(--p4-dark-green-800);
```

Comment:
Maybe here we can use the new `--table-header-gp-green--...` variables like for the Table block?